File: 2003/IntroToVBNETPart2/Presentation/CoffeeClanMember.vb
Imports System

Public Class CoffeeClanMember
    Shared memberCount As Integer = 1
    Dim memberNumber As Integer = memberCount
    Public Sub New()
        memberCount = memberCount + 1
    End Sub
    Public Sub MyCoffeeIsReady()
        Console.WriteLine("Coffee Clan Member #" & memberNumber & ": My Coffee Is Ready")
    End Sub
End Class

File: 2003/IntroToVBNETPart2/Presentation/CoffeeClanMember.cs
using System ;

public class CoffeeClanMember
{
	static int memberCount  = 1 ;
	int memberNumber = memberCount ;
	public CoffeeClanMember () 
	{
		memberCount = memberCount + 1 ;
	}
	public void MyCoffeeIsReady()
	{
		Console.WriteLine("Coffee Clan Member #" + memberNumber + ": My Coffee Is Ready") ;
	}
}
